### PR TITLE
Feature/atr 648 dev create acuminator 3 rn

### PR DESCRIPTION
**Pull request by EAndrosova**
Please review release notes for Acuminator 3.0 and related changes in the documentation

### CODE REVIEW

File: docs/diagnostics/PX1073.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ To prevent the error from occurring, you should remove the code that throws the
 
 This diagnostic is displayed as a warning if the **Enable additional diagnostics for ISV Solution Certification** option (in **Tools > Options > Acuminator > Code Analysis**) is set to `False`.
 
+The diagnostic does not show an error if the following exceptions and their descendants thrown in the `RowPersisted` event handler of a processing graph:
```
Comment (SENya1990) [inline docs/diagnostics/PX1073.md:24]: I'm not sure, but shouldn't it be `are thrown`?
```suggestion
The diagnostic does not show an error if the following exceptions and their descendants are thrown in the `RowPersisted` event handler of a processing graph:
```

File: src/Acuminator/Acuminator.Analyzers/Resources.resx [hunk 2]
```diff
@@ -163,7 +163,7 @@
     <value>Exclude the class from the API Reference</value>
   </data>
   <data name="PX1007Title" xml:space="preserve">
-    <value>A public {0} should have a description in the summary XML tag</value>
+    <value>The {0} DAC or DAC extension should have a description in the summary XML tag</value>
```
Comment (SENya1990) [inline src/Acuminator/Acuminator.Analyzers/Resources.resx:166]: No mentioning of DAC properties

File: docs/ReleaseNotes.md [hunk 3]
```diff
@@ -1,6 +1,54 @@
...
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 3.0
+Acuminator 3.0 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 3.0, the following enhancements have been implemented:
+ - Acuminator now supports Visual Studio 2022 alongside the two previous versions of Visual Studio, 2017 and 2019. 
+ - The [PX1007](diagnostics/PX1007.md) diagnostic now checks only DACs and DAC extensions.
+ - The [PX1008](diagnostics/PX1008.md) diagnostic now validates delegates passed to the `PXLongOperation.StartOperation()` method. A developer can now capture references to the graph instance in delegate closures.
```
[the remark this answers is not in the document]
Comment (EAndrosova) [reply docs/ReleaseNotes.md:11]: Regarding the .snag files: We usually store this type of files alongside .png to be able to edit them later in our SnagIt editor. If you think it's not very appropirate to store them here, I will save them in our Docs repository.

[the remark this answers is not in the document]
Comment (SENya1990) [reply docs/ReleaseNotes.md]: They can be stored here but I'm not sure that we need them. I mean these are Release Notes and screenshots are used only once for them.  It's not an official documentation for Code Map that can be enhanced and reworked several times. We haven't change older release notes AFAIK. 

Do we really plan to change something in them in the future?

File: docs/ReleaseNotes.md [hunk 4]
```diff
@@ -6,48 +6,47 @@ Acuminator 3.0 includes the bug fixes and enhancements described in this section
...
-   Examples of the **Initialization and Activation** node are shown in the following screenshot.
-   ![The Initialization and Activation node for graphs and graph extensions](images/CodeMap_InitAndActive.png)
+ - The **Initialization and Activation** node has been introduced. It displays the following information:
+	- Graph instance and static constructors for graphs and graph extensions
+	- The `IsActive` method for a graph extension and a DAC extension 
+   See the following screenshots for examples of the **Initialization and Activation** node.<br/>
+   ![The Initialization and Activation node for graphs and graph extensions](images/CodeMap_InitAndActive.png)<br/>
    ![The Initialization and Activation node for DACs and DAC extensions](images/CodeMap_DACExtension_InitAndActive.png)
- 
+ - The Code Map windows is automatically reopened at Visual Studio startup if the Code Map was opened at the last Visual Studio closure.s
```
Comment (SENya1990) [inline docs/ReleaseNotes.md:30]: Seems like there is a mistype here
```suggestion
 - The Code Map windows is automatically reopened at Visual Studio startup if the Code Map was opened at the last Visual Studio closure.
```

File: docs/ReleaseNotes.md [hunk 5]
```diff
@@ -6,48 +6,47 @@ Acuminator 3.0 includes the bug fixes and enhancements described in this section
...
  
 ### Fixed Bugs
 In this version of Acuminator, the following bugs have been fixed:
- - Acuminator refactorings did not work when Visual Studio was configured to perform static code analysis in a separate process.
- - The [PX1088](diagnostics/PX1088.md) diagnostic has been removed because its function is covered by the [PX1008](diagnostics/PX1008.md) diagnostic.
- - The [PX1073](diagnostics/PX1073.md) diagnostic now does not show an error if the following exceptions and their descendants thrown in the `RowPersisted`		 event handler of a processing graph:
+ - Acuminator refactorings did not work when Visual Studio was configured to perform static code analysis in a separate process.s
```
Comment (SENya1990) [inline docs/ReleaseNotes.md:34]: Looks like a mistype
```suggestion
 - Acuminator refactorings did not work when Visual Studio was configured to perform static code analysis in a separate process.
```

File: docs/ReleaseNotes.md [hunk 6]
```diff
@@ -1,6 +1,54 @@
...
+ - The new **Initialization and Activation** node has been introduces that displays the following information:
+	- Graph instance and static constructors for graphs, DACs and their extensions
+	- The `IsActive` method is now shown for a graph extension and a DAC extension. 
+	
+   Examples of the **Initialization and Activation** node are shown in the following screenshot.
+   ![The Initialization and Activation node for graphs and graph extensions](images/CodeMap_InitAndActive.png)
+   ![The Initialization and Activation node for DACs and DAC extensions](images/CodeMap_DACExtension_InitAndActive.png)
+ 
+ 
+### Fixed Bugs
```
[the remark this answers is not in the document]
Comment (SENya1990) [reply docs/ReleaseNotes.md:32]: @EAndrosova the bugfix [ATR-621](https://jira.acumatica.com/browse/ATR-621) that fixed support of VS previews is still not described.

**Review by SENya1990 — APPROVED** [review]

**Pull request merged**